src/libostree/ostree-repo-commit.c

```diff
@@ -1624,9 +1624,17 @@ ostree_repo_prepare_transaction (OstreeRepo     *self,
     self->txn.max_blocks = bfree - self->reserved_blocks;
   else
     {
```

Comment:
We should set `self->txn.max_blocks` to **some** value in the else block right? Otherwise the code that uses it will be reading uninitialized memory, I think